jx-requirements.yml

```diff
@@ -4,6 +4,7 @@ cluster:
   project: ""
   provider: gke
   zone: ""
+  environmentGitPublic: false
```

Comment:
Could you move this up to be right below `environmentGitOwner`? So that the ordering is consistent with how we end up marshaling the requirements during `jx boot` and friends - it wants to be in alphabetical order then. =)